js/index.js

```diff
@@ -1,9 +1,12 @@
+let spaceship;
```

Comment:
Variables should use only var to support old browsers or the code should be transpiled using browserify.
Also all global variables should be in variables file